cache_pdks/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Install PDKs.
+
+inputs:
+  pdk_root:
+    description: PDK root to install files.
+    required: true
+  ol_path:
```

Comment:
Does this actually need to be part of the inputs? Since openlane is just one way to get/build a particular open_pdks version, it doesn't necessarly need to bleed in the action contract.

Reply:
The goal of these actions is for use with OpenLane, see also my comment above about magic version. For these reasons, I think it's ok to keep ol_path.

Reply:
I have changed to use only `ol_tag`.